test/integration/socketio.test.js

```diff
@@ -132,4 +132,14 @@ describe('Socket.io client authentication', () => {
       });
     });
   });
+
+  it.only('handles upgrade events', done => {
```

Comment:
@marshallswain @daffl do you know how to simulate upgrade events? I can't get this test to pass either but I don't think it is actually testing upgrade functionality anyway.